### PR TITLE
Fix error message

### PR DESCRIPTION
**Pull request by digama0**
I'm not entirely sure if the message is still correct, it seems to have survived a number of refactors, but it is mangled english anyway.

### CODE REVIEW

File: crates/hir_expand/src/db.rs [hunk 1]
```diff
@@ -366,7 +366,7 @@ fn macro_expand(db: &dyn AstDatabase, id: MacroCallId) -> ExpandResult<Option<Ar
 
     let macro_arg = match db.macro_arg(id) {
         Some(it) => it,
-        None => return ExpandResult::str_err("Fail to args in to tt::TokenTree".into()),
+        None => return ExpandResult::str_err("Fail to lower args to token tree".into()),
```
Comment (lnicola) [inline crates/hir_expand/src/db.rs:369]:
```suggestion
        None => return ExpandResult::str_err("Fail to lower macro args to token tree".into()),
```

Comment (lnicola) [reply crates/hir_expand/src/db.rs:369]: Also, `Failed to find macro definition` a couple of lines below.

**Comment by matklad** [comment]
Yeah, I think that's the right message, thanks!

bors r+

**Comment by lnicola** [comment]
bors r+

changelog internal (first contribution) fix some error messages

**Pull request merged**